### DISCUSSION TITLE
ar/hub/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#19001] [Docs]
# ar/hub/

اكتشف Ultralytics HUB، أداة الويب المتكاملة لتدريب ونشر نماذج YOLOv5 و YOLOv8 . ابدأ بسرعة مع النماذج المدربة مسبقًا والميزات سهلة الاستخدام.

https://docs.ultralytics.com/ar/hub/

**Comment by UltralyticsAssistant** [comment]
👋 Hello, thank you for your interest in Ultralytics 🚀! It seems you're exploring the **Ultralytics HUB**, our comprehensive web-based platform for training and deploying YOLOv5 and YOLOv8 models. That's fantastic! 🌟 For new users, we highly recommend checking out the [Ultralytics HUB Documentation](https://docs.ultralytics.com/ar/hub/), where you'll find step-by-step guides and all the resources to get started quickly with pre-trained models and user-friendly features.

If you're reporting a 🐛 Bug or facing any specific issues, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) that demonstrates the problem. This will help us debug it more effectively.

If you have any ❓ Questions about using the **HUB** or custom training workflows, feel free to share more details! Please include screenshots, error logs, or any other relevant context to streamline assistance. Additionally, you can reference our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) for optimizing your workflows.

Join the Ultralytics community where it suits you best! For real-time support and discussions, hop on over to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer more structured conversations? Visit our [Discourse](https://community.ultralytics.com/). And for informal sharing and discussion, check out our [Subreddit](https://reddit.com/r/Ultralytics).

## Upgrade

If you're using the `ultralytics` package locally, make sure you upgrade to the latest version to ensure you're equipped with all the recent features and fixes. Run the command below in your terminal:

```bash
pip install -U ultralytics
```

This will update `ultralytics` along with its dependencies. Verify you're running it in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/).

## Environments

You can also run YOLO and explore its functionalities in several easy-to-use environments with pre-configured settings:

- **Notebooks**: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM: [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/).
- **Amazon** Deep Learning AMI: [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/).
- **Docker Image**: [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests ensure reliability across YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu.

This is an automated response to assist you promptly, and an Ultralytics engineer will also respond soon to further support you. Feel free to share additional details in the meantime. Happy exploring! 🚀
